client/sandbox/react-nextjs/pages/play/linkunlink_ruleparams.tsx

```diff
@@ -0,0 +1,203 @@
+import { useState } from 'react';
+import { init, tx, id, i } from '@instantdb/react';
+import config from '../../config';
+import EphemeralAppPage from '../../components/EphemeralAppPage';
+
+const schema = i.schema({
+  entities: {
+    groups: i.entity({
+      name: i.string(),
+      ownerId: i.string(),
+    }),
+    users: i.entity({
+      name: i.string(),
+    }),
+  },
+  links: {
+    groupMembers: {
+      forward: {
+        on: 'groups',
+        has: 'many',
+        label: 'members',
+      },
+      reverse: {
+        on: 'users',
+        has: 'many',
+        label: 'groups',
+      },
+    },
+  },
+});
+
+const perms = {
+  groups: {
+    allow: {
+      view: 'true',
+      create: 'true',
+      update: 'true',
```

Comment:
cc @tonsky

QQ: If I set `update` to false, linking and unlinking will fail. 

Is the rule something like: 
For a `link` or `unlink`, the `update` check has to pass, as well as the `link` check?

Reply:
Tried to explain it here https://github.com/instantdb/instant/pull/1514/files#diff-85bc61244af9920daebe28894bbc3f5a33e8df53f127fc5b7e243471784ea027

Reply:
If you set update on groups to false, it should not matter, yes

Reply:
Okay, tested once more...and you are totally right. Maybe my git foo had me in a bad state before. Thanks!